perceval/backend.py

```diff
@@ -327,6 +328,11 @@ def __init__(self, categories, from_date=False, to_date=False, offset=False,
                                type=int, default=0,
                                help="offset to start fetching items")
 
+        if blacklist:
+            group.add_argument('--blacklist-ids', dest='blacklist_ids',
+                               nargs='*', type=str,
```

Comment:
Does this make sense to be a list of strings?

Reply:
I guess so, since it is more inclusive, however it may not be applicable for all backends (gitlab, github which IDs are numerics). An example of non-numeric IDs are commit hashes. Maybe we can re/define the `--blacklist-ids` for each backend.

Reply:
There are no many backends with a blacklist: jenkins, gerrit and gitlab so far. So, it's possible to use the same parameter. My main concern is to make clear what `ids` mean for each case.

Reply:
A new proposal is available at: https://github.com/chaoss/grimoirelab-perceval/pull/575. The idea is to compare the approaches and decide how to proceed.